scrapy/core/downloader/tls.py

```diff
@@ -48,6 +48,11 @@ def _identityVerifyingInfoCallback(self, connection, where, ret):
                         'Remote certificate is not valid for hostname "{}"; {}'.format(
                             self._hostnameASCII, e))
 
+                except ValueError as e:
+                    logger.warning(
+                        'SSL/TLS verification failed for hostname "{}"; {}'.format(
+                            self._hostnameASCII, e))
```

Comment:
@kmike , how about `'Ignoring remote certificate verification failure for hostname "{}"; {}'.format(self._hostnameASCII, e))`, without the SSL mention?